glib/src/date.rs

```diff
@@ -26,7 +26,7 @@ unsafe impl Sync for Date {}
 
 impl Date {
```

Comment:
Should be marked as renamed on gir's side as well

Reply:
It's not listed at all in `Gir.toml` and if it was it wouldn't have much effect because we don't inject docs in `glib`. There are lots of types missing from `Gir.toml`.

Reply:
Added this one for now